openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -1027,6 +1028,8 @@
     "data-insight-message": "Manage data insights pipeline.",
     "data-insight-page-views": "Displays the number of times a dataset type was viewed.",
     "data-insight-pipeline-description": "Monitor data usage and set up KPIs by deploying the data insights pipeline. Refer to our documentation to learn more <0>{{link}}</0>",
+    "data-insight-report-send-failed-message": "Failed to send data insight report!",
+    "data-insight-report-send-success-message": "Data Insight Report sent successfully!",
```

Comment:
Let's remove "!" 
cc: @ShilpaVernekar

Reply:
Sure, can we take care of this in another patch?

Reply:
@chirag-madlani Yeah doesn't make sense to use "!" for a failed to send report. But to just add a full stop.